src/commands/registries/azure/tasks/scheduleRunRequest.ts

```diff
@@ -3,17 +3,19 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { ContainerRegistryManagementClient, DockerBuildRequest as AcrDockerBuildRequest, FileTaskRunRequest as AcrFileTaskRunRequest, OS as AcrOS, Run as AcrRun } from "@azure/arm-containerregistry"; // These are only dev-time imports so don't need to be lazy
-import { IActionContext, IAzureQuickPickItem, nonNullProp } from '@microsoft/vscode-azext-utils';
+import type { DockerBuildRequest as AcrDockerBuildRequest, FileTaskRunRequest as AcrFileTaskRunRequest, OS as AcrOS, Run as AcrRun, ContainerRegistryManagementClient } from "@azure/arm-containerregistry"; // These are only dev-time imports so don't need to be lazy
+import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
```

Comment:
Is `scheduleRunRequest` still being used? If so we should lazy-load azureutils

Reply:
yeah, we still use it for `buildImageInAzure`

Reply:
actually we can use the `getResourceGroupFromAzureRegistryItem()` from `registryTreeUtils.ts`, so we don't even need to import "@microsoft/vscode-azext-azureutils"